jmclient/jmclient/client_protocol.py

```diff
@@ -29,6 +20,7 @@
                         load_program_config, get_log, get_p2sh_vbyte,
                         RegtestBitcoinCoreInterface)
 from jmbase import _byteify
+from jmdaemon import start_daemon
```

Comment:
Want to avoid this unconditional import here, so jmclient doesn't have a jmdaemon dependency by default; can you put it in the try/except block in `if daemon:` in `start_reactor`?